static/app/views/replays/detail/network/index.tsx

```diff
@@ -237,23 +252,25 @@ const StyledPanelTable = styled(PanelTable)<{columns: number}>`
     border-radius: 0;
     color: ${p => p.theme.subText};
     line-height: 16px;
+    text-transform: none;
+
+    ${/* sc-selector */ UnstyledButton} {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
```

Comment:
Instead of nesting this, we should create a new styled component:

```suggestion
const UnstyledHeaderButton = styled(UnstyledButton)`...`
```

I would do the same with `PanelTableHeader` too.

Reply:
Done! for `PanelTableHeader ` I think I cannot do the same as this component is being used inside the `PanelTable` component

Reply:
Ah right